netdata-installer.sh

```diff
@@ -1151,7 +1260,7 @@ fi
 # -----------------------------------------------------------------------------
 progress "Install netdata"
 
-if ! run $make install; then
+if ! run ${cmake} --install "${NETDATA_BUILD_DIR}"; then
```

Comment:
@Ferroin 

`--install` was first added in 3.15:

https://cmake.org/cmake/help/v3.15/manual/cmake.1.html#cmdoption-cmake-install

Building from source will fail on Debian 10, because:

```
$ cmake --vesion
cmake version 3.13.4
```

We might be able to get away with using `ninja install` or `make install`. However, the proper fix is to require cmake >= `3.15`.